### PR TITLE
libcudd: update to 3.0.0; gr1c: update to 0.13.2

### PR DESCRIPTION
**Pull request by slivingston**
#### Description

* Change the maintainer email addresses: my old SCL address is defunct, but the slivingston address is operational and will continue to be.
* Update to the most recent release of CUDD, which includes a `configure` script such that the patch files associated with the Portfile are not necessary.
* Update to the most recent release of gr1c, and change the Portfile to fetch releases from GitHub; the patch files are not necessary anymore.

Note that the upstream website for CUDD has been offline for years now. It was last captured by the Internet Archive 6 years ago: <https://web.archive.org/web/20180217063726/http://vlsi.colorado.edu/~fabio/CUDD/>. Since then, the upstream release version 3.0.0 has been available at the mirror used in this pull request.

As the maintainer of these portfiles, I am so delayed because I was without a Mac until recently.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 13.5.2 22G91 arm64
Xcode 15.2 15C500b

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: devel/libcudd/Portfile [hunk 1]
```diff
@@ -3,10 +3,10 @@
 PortSystem          1.0
 
 name                libcudd
-version             2.5.0
+version             3.0.0
 categories          devel
 platforms           darwin
-maintainers         caltech.edu:SCL openmaintainer
+maintainers         cds.caltech.edu:slivingston openmaintainer
```
Comment (herbygillot) [inline devel/libcudd/Portfile:9]: You should include your Github handle:

```suggestion
maintainers         {cds.caltech.edu:slivingston @slivingston} \
                    openmaintainer
```

**Review by barracuda156 — APPROVED** [review]
I can confirm that both ports build fine for me now on 10.6.

File: math/gr1c/Portfile [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+PortGroup           github 1.0
+
+github.setup        tulip-control gr1c 0.13.2 v
 
-name                gr1c
-version             0.9.0
 categories          math devel
 platforms           darwin
-maintainers         caltech.edu:SCL openmaintainer
+maintainers         cds.caltech.edu:slivingston openmaintainer
```
Comment (herbygillot) [inline math/gr1c/Portfile:10]: Sorry, missed this, your Github handle should be here as well

Comment (slivingston) [reply math/gr1c/Portfile]: My fault: I should have generalized your previous comment. Thanks for reviewing this!

**Pull request merged**